src/CattyTests/Resources/ParserTests/Custom/ValidProjectAllBricks/ValidProjectAllBricks0991.xml

```diff
@@ -656,6 +656,15 @@
               <commentedOut>false</commentedOut>
               <spinnerSelection>80</spinnerSelection>
             </brick>
+            <brick type="SetBackgroundByIndexBrick">
+              <commentedOut>false</commentedOut>
+              <formulaList>
+                <formula category="SetBackgroundByIndexBrick">
```

Comment:
This is an invalid XML. There should be two bricks in it with the category named (a) `LOOK_INDEX` and (b) `BACKGROUND_INDEX`.